config/config.txt

```diff
@@ -285,7 +288,10 @@ GITHUBURL https://www.github.com/beestation/beestation-hornet
 DONATEURL https://www.patreon.com/user?u=10639001
 
 ## Discord Invite
-DISCORDURL https://discord.gg/zUe34rs
+DISCORDURL https://discord.gg/ss13
```

Comment:
I updated this link since the previous one used for example purposes was invalid and could potentially be overwritten by some random discord server.

Reply:
It's arguably better to use a tracked invite link here to see how many joins come from this config entry, but also.... I'm changing it to be the website in config anyways, so _eh_